localization/i18n/es/OrcaSlicer_es.po

```diff
@@ -961,10 +961,10 @@ msgid "points"
 msgstr "puntos"
 
 msgid "Revert gap between characters"
-msgstr "Revert gap between characters"
+msgstr "Revertir el espacio entre caracteres"
```

Comment:
"Deshacer" migh be a better word here.

Reply:
There were similar and changed all of them. Thanks.